app/src/main/java/ch/epfl/sweng/swenggolf/offer/ShowOfferActivity.java

```diff
@@ -57,7 +57,9 @@
 import static ch.epfl.sweng.swenggolf.Permission.GPS;
 import static ch.epfl.sweng.swenggolf.location.AppLocation.checkLocationPermission;
 
-
+/**
+ * Fragment which shows the offers.
```

Comment:
Reading the javadoc we could think it shows multiple offers or a list of offers. You could replace it by something like "shows an offer".